Dockerfile

```diff
@@ -13,23 +16,16 @@ RUN useradd -m -u $UID -g $GID -s /bin/bash murphy
 
 # set up requirements
 WORKDIR /home/murphy
-ADD --chown=murphy:murphy ./requirements.txt /home/murphy/requirements.txt
+ADD --chown=murphy:murphy ./requirements.txt .
 RUN pip install -r /home/murphy/requirements.txt
 
-# set up strider
-ADD --chown=murphy:murphy ./strider /home/murphy/strider
-ADD --chown=murphy:murphy ./setup.py /home/murphy/setup.py
-RUN pip install -e .
-
-# get meta-stuff
-ADD --chown=murphy:murphy ./logging_setup.yml /home/murphy/logging_setup.yml
-ADD --chown=murphy:murphy ./run_workers.py /home/murphy/run_workers.py
-ADD --chown=murphy:murphy ./main.sh /home/murphy/main.sh
+# Copy in files
+ADD --chown=murphy:murphy . .
```

Comment:
There is a bunch of stuff here that the container doesn't need, like /examples and /tests. None of that is huge, but could we exclude them simply by adding to .dockerignore?